src/pages/CardPage.tsx

```diff
@@ -1,6 +1,102 @@
+import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
+//import Footer from "@/components/Footer"; // Already renders, figuring out if I need to add a declarions file later and adjust main rendering. not prio, just annoying to see
+import { cn } from "@/lib/utils";
+import "./CardPage.css";
+//import React from "react";
+
+interface Card {
+  id: number;
+  name: string;
+  descriptionRaw: string;
+  attack: number;
+  cost: number;
+  health: number;
+  flavorText: string;
+  artistName: string;
+  cardCode: string;
+  collectible: boolean;
+  set: string;
+  assets: {
+    gameAbsolutePath: string;
+    fullAbsolutePath: string;
+  }[];
+  formats: string[];
+}
```

Comment:
I noticed that some of these don't match what the API gives us so they aren't working.
ex: `artistName` should be `artist_name`